frontend/public/components/network-policy.jsx

```diff
@@ -88,31 +89,38 @@ const IngressRow = ({ingress, namespace, podSelector}) => {
   </div>;
 };
 
-const Details = ({obj: np}) => <React.Fragment>
-  <div className="co-m-pane__body">
-    <SectionHeading text="Namespace Overview" />
-    <ResourceSummary resource={np} podSelector={'spec.podSelector'} showNodeSelector={false} />
-  </div>
-  <div className="co-m-pane__body">
-    <SectionHeading text="Ingress Rules" />
-    <p className="co-m-pane__explanation">
-      Pods accept all traffic by default.
-      They can be isolated via Network Policies which specify a whitelist of ingress rules.
-      When a Pod is selected by a Network Policy, it will reject all traffic not explicitly allowed via a Network Policy.
-      See more details in <a target="_blank" rel="noopener noreferrer" href="https://kubernetes.io/docs/concepts/services-networking/network-policies/">Network Policies Documentation</a>.
-    </p>
-    {
-      _.isEmpty(_.get(np, 'spec.ingress[0]', [])) ?
-        `All traffic is allowed to Pods in ${np.metadata.namespace}.` :
-        <div className="co-m-table-grid co-m-table-grid--bordered">
-          <IngressHeader />
-          <div className="co-m-table-grid__body">
-            { _.map(np.spec.ingress, (ingress, i) => <IngressRow key={i} ingress={ingress} podSelector={np.spec.podSelector} namespace={np.metadata.namespace} />) }
+const Details_ = ({flags, obj: np}) => {
+  const NetworkPolicyDocs = flags[FLAGS.OPENSHIFT]
+    ? 'https://docs.openshift.com/container-platform/3.10/admin_guide/managing_networking.html#admin-guide-networking-networkpolicy'
```

Comment:
This needs to use the configured base documentation URL from 

https://github.com/openshift/console/blob/master/frontend/public/components/utils/documentation.tsx

I would export HELP_TOPICS there and add this as a topic.